src/main/java/net/snowflake/client/jdbc/cloud/storage/SnowflakeS3Client.java

```diff
@@ -4,6 +4,7 @@
 
 package net.snowflake.client.jdbc.cloud.storage;
 
+import static net.snowflake.client.core.Constants.SESSION_EXPIRED_GS_CODE;
```

Comment:
I like the new added error because it's more specific to what the problem is! It's still a SnowflakeSQLException so I think it's okay to change the message sligthly.